app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
@deviantony I think that users can use this url as a bookmark or as #833 requested. 

Why would we want to prevent this usage from users?